Formula/libjwt.rb

```diff
@@ -0,0 +1,37 @@
+class Libjwt < Formula
+  desc "JSON Web Token C library"
+  homepage "https://github.com/benmcollins/libjwt"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "check" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "openssl"
+
+  def install
+    system "autoreconf", "-v", "-i"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "all"
```

Comment:
Is "all" not the default make target?

Reply:
The Makefile is generated by autotools and the `all` has a dependency `all: all-recursive`. It must be `make all`, otherwise some dependencies will not be met.

Reply:
I think you can just run `make` no?